prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/FhirPathUtilsTests.kt

```diff
@@ -88,4 +102,92 @@ class FhirPathUtilsTests {
         assertThat(result[0]).isInstanceOf(ServiceRequest::class.java)
         assertThat((result[0] as ServiceRequest).id).isEqualTo(servRequest.id)
     }
+
+    @Test
+    fun `test evaluateString`() {
+        val bundle = Bundle()
+        bundle.timestamp = Date()
+
+        val observation = Observation()
+        observation.effective = DateTimeType("2015-04-11T12:22:01-04:00")
+        bundle.addEntry().resource = observation
+
+        // Test timestamp of java Date
+        var path = FhirPathUtils.parsePath("Bundle.timestamp")
+        var result = FhirPathUtils.evaluateString(null, bundle, bundle, path!!)
+        assertThat(result).isNotEmpty()
+        assertThat(result).isInstanceOf(String::class.java)
```

Comment:
You do not have to test this.  This is controlled by the return type of the function.